setup.py

```diff
@@ -16,7 +16,7 @@
     install_requires=[
         "reportportal-client",
         "robotframework==4.*",
-        "robotframework-debuglibrary",
+        "robotframework-debuglibrary>=2.0.0",
```

Comment:
Is this related to the error that we see in the logs recently?

Reply:
yes

Reply:
shall we remove the library?